release/likely-commonjs.js

```diff
@@ -826,7 +826,10 @@ var LikelyButton = function () {
         value: function initCounter() {
             var options = this.options;
 
-            if (options.counters && options.counterNumber) {
+            if (!options.counters) {
+                return;
+            }
+            if (options.counterNumber) {
                 this.updateCounter(options.counterNumber);
```

Comment:
**Before**

- `options.counters` is false
- the first condition is not met, that's fine
- then we go else if and since `options.counterUrl` is still there we go into this branch and run `updateCounters` (which shows them)!

Here's the bug.

**After**

- if counters are disabled that's it, we don't go any further at all

Reply:
gotcha, thanks for addressing this